test/config-next/zlint.toml

```diff
@@ -15,6 +15,10 @@ ignore_lints = [
   # issued under the "classic" profile, but have removed it from our "tlsserver"
   # and "shortlived" profiles.
   "pkilint:cabf.serverauth.subscriber_rsa_digitalsignature_and_keyencipherment_present",
+  # Some linters continue to complain about the lack of an AIA OCSP URI, even
+  # when a CRLDP is present.
```

Comment:
This is unfortunate, perhaps we should file issues for these two.

Reply:
Good call, thanks! https://github.com/certlint/certlint/issues/20

x509lint hasn't been modified in 5 years, so I'm not going to bother filing an issue on that one.